test/e2e-v2/script/docker-compose/base-compose.yml

```diff
@@ -0,0 +1,102 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '2.1'
+
+services:
+  oap:
+    image: skywalking/oap:latest
+    expose:
+      - 11800
+      - 12800
+      - 10051
+      - 5005
+    networks:
+      - e2e
+    volumes:
+      - ./../prepare/setup-oap/download-mysql.sh:/download-mysql.sh
+      - ./../prepare/setup-oap/log4j2.xml:/skywalking/config/log4j2.xml
+    environment:
```

Comment:
should we set the mysql and these environment variables in base-comose?

Reply:
Other e2e tests may need to use this configuration and use it in conjunction with the service in base-compose, so I think it’s better to put it there.

Reply:
It's OK to set these here (as long as they are reasonable in our E2E), they serve as the default configurations in E2E cases, without modifying the original `application.yaml`

Reply:
@wankai123 I am a little confused about your asking. What setup? JDBC connection URL?